cli/flox-rust-sdk/src/providers/catalog.rs

```diff
@@ -144,7 +144,7 @@ impl CatalogClient {
         }
 
         let client = {
-            let timeout = std::time::Duration::from_secs(15);
+            let timeout = std::time::Duration::from_secs(60);
```

Comment:
Is there a server side timeout?

Reply:
Not that I can reasonably discern.  This is the timeout that @nicklewis pointed me to when asked about the issue.

From what I can tell Postgres has no default statement timeout, and we persist connections.  There is likely some connection timeout that would trigger if the connection between RDS and the service died, but that's a different scenario.

We can set a per statement timeout, which may be a good idea, but we don't currently do that.